src/app/page.tsx

```diff
@@ -1,24 +1,61 @@
-import { CategoriesExpandableList } from "@/components/CategoriesExpandableList";
-import { CategoriesMenu } from "@/components/CategoriesMenu";
-import Header from "@/components/Header";
+"use client";
+
+import { useState } from "react";
+import dynamic from "next/dynamic";
+
+import CategoriesExpandableList from "@/components/CategoriesExpandableList";
 import ListItemCard from "@/components/ListItemCard";
+import { Modal } from "flowbite-react";
+import PreviewItemsAddedSkeleton from "@/components/PreviewItemsAdded/PreviewItemAddedSkeleton";
+
+const PreviewItemAdded = dynamic(
+  () =>
+    import("@/components/PreviewItemsAdded/PreviewItemAdded").then(
+      (module) => module.default
+    ),
+  {
+    loading: () => <PreviewItemsAddedSkeleton />,
+  }
+);
+
+const items = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
 export default function Home() {
-  const items = [1,2,3,4,5,6,7,8,9,10]
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const handleAddToCart = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleOnModalClose = () => {
+    setIsModalOpen(false);
+  };
+
   return (
-    <>
-      <div className="flex justify-center pb-4">
-        <div className="pt-3 flex flex-row w-3/4">
-          {/* <div className="bg-red-500 w-96">
-            Hierarchy
-          </div> */}
-          {/* <CategoriesMenu/> */}
-          <CategoriesExpandableList/>
-          <div className="flex flex-col w-full gap-2">
-            {items.map(item => <ListItemCard className="rounded-md" key="{item}"/>)}
-          </div>
+    <div className="flex justify-center pb-4">
+      <div className="flex lg:min-w-[1024px] md:max-w-[768px] gap-2 pt-3">
+        <CategoriesExpandableList />
+        <div className="flex flex-col w-full gap-2">
+          {items.map((item) => (
+            <ListItemCard
+              className="rounded-md"
+              key={item}
```

Comment:
Watch out for `key="{item}"` vs `key={item}`. Even more, would be indicated to add `key={item.id}` once the data is clear